packages/react-native/Libraries/Text/RCTTextAttributes.mm

```diff
@@ -8,16 +8,85 @@
 #import <React/RCTTextAttributes.h>
 
 #import <React/RCTAssert.h>
+#import <React/RCTCursor.h> // [macOS]
 #import <React/RCTFont.h>
 #import <React/RCTLog.h>
 
-#if TARGET_OS_OSX // [macOS
-#import <React/RCTCursor.h>
-#endif // macOS]
-
 NSString *const RCTTextAttributesIsHighlightedAttributeName = @"RCTTextAttributesIsHighlightedAttributeName";
 NSString *const RCTTextAttributesTagAttributeName = @"RCTTextAttributesTagAttributeName";
 
+#if TARGET_OS_OSX // [macOS
+// Duplicate this function here as RCTTextAttributes doesn't depend on RCTView
+static NSCursor *NSCursorFromRCTCursor(RCTCursor cursor)
+{
+  NSCursor *platformCursor;
+
+  switch (cursor) {
+    case RCTCursorAlias:
+      platformCursor = [NSCursor dragLinkCursor];
+      break;
```

Comment:
Why not just return instead of using a local variable?
```suggestion
    case RCTCursorAlias:
      return [NSCursor dragLinkCursor];
```

Reply:
Holdover from my old (current?) teams coding guidelines: https://microsoft.github.io/objc-guide/MethodsAndImplementations/EarlyReturns.html

Reply:
I don't agree with it in this case. It's a simple converter function. If it gets more complicated than that, it's wrong in any case? But I don't have a super strong opinion here.

Reply:
This matches the style I see elsewhere in `conversions.h` (not this file I know), so I can update. That specific rule was always one of our more controversial ones in code reviews :D